Sources/NIOSSL/SwiftCrypto/Zeroization_boring.swift

```diff
@@ -0,0 +1,26 @@
+//===----------------------------------------------------------------------===//
```

Comment:
We can just call this `Zeroization` and drop the `_boring`.

Reply:
Done in 8007b79